src/main/java/core/basesyntax/dto/OperationValidator.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.exceptions.InvalidOperationException;
+
+public class OperationValidator {
```

Comment:
I assume this class it's never used because you haven't created || removed your main method?